ch2-inventory/student-trungdc/03-hostgroups-children.yml

```diff
@@ -0,0 +1,19 @@
+all:
```

Comment:
we don't need to EXPLICITLY have all/ungroup group defined. you can remove them to make your inventory shorter and more readable